graph/ops_convolve.go

```diff
@@ -133,72 +94,38 @@ func (conv *ConvolutionBuilder) ChannelsAxis(channelsAxisConfig timage.ChannelsA
 	return conv
 }
 
-// AxesConfig specify the exact configuration of the axes on the input (x/input and kernel) and output of
-// the Convolve operation. This is advanced (and may not be supported in every backend), but it's powerful.
-// Consider using `ConvolutionBuilder.ChannelsAxis` instead.
-//
-// The default is `ChannelsAxis(timage.ChannelsLast)`.
 func (conv *ConvolutionBuilder) AxesConfig(axes ConvolveAxesConfig) *ConvolutionBuilder {
 	conv.axes = axes
 	return conv
 }
 
-// Strides sets the strides of the convolution. It sets the same value for every dimension.
-// The default is 1.
-//
-// The stride is how many steps to move after a convolution. A value of 2 will halve the input
-// size, since a convolution will be done at every other position, and so on. It can be defined
-// separately per dimension.
-//
-// One cannot use strides and dilation at the same time.
 func (conv *ConvolutionBuilder) Strides(strides int) *ConvolutionBuilder {
 	perDim := xslices.SliceWithValue(conv.numSpatialDims, strides)
 	return conv.StridePerDim(perDim...)
 }
 
-// StridePerDim sets the strides for each spatial dimension of the convolution.
-// The default is 1 for every dimension.
-//
-// The stride is how many steps to move after a convolution.
-// A value of 2 will halve the input size, since a convolution will be done at every other position, and so on.
-// It can be defined separately per dimension.
-//
-// One cannot use strides and dilation at the same time.
 func (conv *ConvolutionBuilder) StridePerDim(strides ...int) *ConvolutionBuilder {
 	if len(strides) != conv.numSpatialDims {
+		log.Println("\nStrides:", strides, "\nLen of strides:", len(strides), "\nConvolutionBuilder:", conv.numSpatialDims)
```

Comment:
If this information is useful, maybe move it to the error message below, in the `Panicf` ?